src/test/java/de/tum/in/www1/artemis/service/FileServiceTest.java

```diff
@@ -42,6 +44,30 @@ class FileServiceTest extends AbstractSpringIntegrationIndependentTest {
     // the resource loader allows to load resources from the file system for this prefix
     private final Path overridableBasePath = Path.of("templates", "jenkins");
 
+    private static final URI VALIDCURRENTBACKGROUNDPATH = URI.create("/api/uploads/images/drag-and-drop/backgrounds/1/BackgroundFile.jpg");
```

Comment:
Can you rename all of these attributes following this scheme? This makes that more readable.
```suggestion
    private static final URI VALID_CURRENT_BACKGROUND_PATH = URI.create("/api/uploads/images/drag-and-drop/backgrounds/1/BackgroundFile.jpg");
```

Reply:
Addressed by [4d90f856a385f6787e592fdcdb13dae79facd8d2](https://github.com/ls1intum/Artemis/pull/7170/commits/4d90f856a385f6787e592fdcdb13dae79facd8d2).